packages/back-end/src/integrations/GoogleAnalytics.ts

```diff
@@ -262,6 +262,10 @@ const GoogleAnalytics: SourceIntegrationConstructor = class
     return this.params.delimiter || ":";
   }
 
+  ensureFloat(col: string): string {
```

Comment:
GA is not a SQL integration and is pretty much deprecated as a direct connection. GA is deprecated in favor of GA4 from Google, and GA4 users flow their data through BigQuery to us.

This code won't ever get used.